haystack/pipelines/base.py

```diff
@@ -1072,7 +1072,7 @@ def eval(
         - `context_matching_...` param to fine-tune the fuzzy matching mechanism that determines whether some text contexts match each other (only affects 'context' scopes, default values should work most of the time)
 
         :param labels: The labels to evaluate on
-        :param documents: List of List of Document that the first node in the pipeline should get as input per multilabel. Can be used to evaluate a pipeline that consists of a reader without a retriever.
+        :param documents: List of Document that the first node in the pipeline should get as input per multilabel. Can be used to evaluate a pipeline that consists of a reader without a retriever.
```

Comment:
Might be confusing whether the parameter is now a list only or a list of lists. I'd suggest to change it to: "List of List of Document, where the input to the first node in the pipeline is one List of Document per MultiLabel"

Reply:
RIght, I think it should only be a List of List of Document objects.